### PR TITLE
Convert desktop tests to use Maze Runner Commands

### PR DESCRIPTION
**Pull request by twometresteve**
## Goal

Convert the desktop tests (macOS, Windows and webGL) to use Maze Runner commands.

## Design

Rather than setting environment variables, the test fixture will make and HTTP request to Maze Runner's `/command` endpoint to determine what it should do next.  The call is made from `Main.Start`.

The eventual aim (not in this PR) will be to combine the mobile tests to use the same mechanism and set of feature files.

## Changeset

- Add call to `/command` and associated processing.
- We now need to make running of the test fixture a non-blocking call.  As part of this I've added a Cucumber step to send a Command to explicitly quit the app, rather than having the app close itself on a timer.
- Remove code relating to setting and processing of environment variables.
- Some scenarios that were duplicated for macOS/webGL have now been combined.
- `PreventCrashPopups` added to the native bundle to try and prevent the app from displaying a popup after crashing multiple times (and breaking all subsequent scenarios).  
 
## Testing

Covered by a full CI run.

### CODE REVIEW

File: features/fixtures/maze_runner/Assets/Scripts/Main.cs [hunk 1]
```diff
@@ -42,37 +98,8 @@ public void Start()
...
-            Bugsnag.ClearMetadata("init");
-            Bugsnag.ClearMetadata("test", "test2");
-        }
-        RunScenario(scenario);
-    }
 
-    void CloseApplication()
-    {
-        Application.Quit();
+        StartCoroutine(RunNextMazeCommand());
```
Comment (twometresteve) [inline features/fixtures/maze_runner/Assets/Scripts/Main.cs:102]: Here...

File: features/fixtures/maze_runner/Assets/Scripts/Main.cs [hunk 2]
```diff
@@ -7,114 +8,109 @@
...
 
-    private float _closeTime = 5;
+#if UNITY_STANDALONE || UNITY_WEBGL
+    private string _mazeHost = "http://localhost:9339";
+#else
+    private string _mazeHost = "http://bs-local.com:9339";
+#endif
 
-    public void Start()
+    IEnumerator RunNextMazeCommand()
```
Comment (rich-bugsnag) [inline features/fixtures/maze_runner/Assets/Scripts/Main.cs:50]: Nit, would be nice if Main.Start was the first method in the file to save future devs time working out entry points etc...

Comment (twometresteve) [reply features/fixtures/maze_runner/Assets/Scripts/Main.cs:68]: Moved.

**Review by rich-bugsnag — APPROVED** [review]
Nice!

**Pull request merged**